\chapter{Materials and Methods}

\hrulefill

\section{Fly husbandry and stock keeping}
The wild-type strains of the following \emph{Drosophila} species were used in all experiments: \emph{D. melanogaster} Oregon-R and \emph{w\textsuperscript{1118}}; \emph{D. simulans w\textsuperscript{[501]}} (reference strain - \url{http://www.ncbi.nlm.nih.gov/genome/200?genome_assembly_id=28534}); \emph{D. yakuba} Cam-115 \citep{coyne_genetic_2004}; \emph{D. pseudoobscura pseudoobscura} (reference strain - \url{http://www.ncbi.nlm.nih.gov/genome/219?genome_assembly_id=28567}). \emph{D. melanogaster, D. simulans} and \emph{D. yakuba} flies were kept at 25°C on standard cornmeal medium. \emph{D. pseudoobscura} flies were kept at 22.5°C in low humidity, on banana-opuntia-malt medium (1000 ml water, 30 g yeast, 10 g agar, 20 ml Nipagin, 150 g mashed banana, 50 g molasses, 30 g malt, 2.5 g opuntia powder). All embryo collections were performed at 25°C with the exception of the \emph{D. pseudoobscura} staged collections for FAIRE-seq, which were performed at 22.5°C. Flies were allowed to lay for varying periods of time on agar plates supplemented with grape juice and streaked with fresh yeast paste.
\paragraph{}
All microinjections to generate transgenic lines were performed by Sang Chan in the Department of Genetics injection facility. Before injections, flies were kept in cages for 2 days at 25°C, with a fresh grape juice-agar plate with yeast paste provided twice a day. After 2 days, the plates were changed every 30 minutes for 2 hours, and then embryos were collected after a 30-minute lay. In an 18°C injection room, embryos were washed and dechorionated in 50\% bleach for 3 minutes. They were then rinsed with cold water, blotted dry on a paper towel and transferred with a paintbrush to a coverslip on which a stripe of heptane-glue had been painted (made by dissolving sellotape in heptane). The embryos were aligned on the heptane-glue with forceps and covered with 10 S Voltalef oil (VWR). The posterior end of each embryo was injected using a glass needle loaded on a Leitz micromanipulator. The injection mix consisted of a piggyBac helper plasmid at 0.4 μg/μl and a piggyBac plasmid containing the construct of interest at 0.6 μg/μl.
\paragraph{}
Injected embryos were transferred on the coverslip to a grape juice-agar plate with a small dot of yeast paste and left to develop for 24 hours at 25°C. \emph{D. pseudoobscura} embryos were allowed to develop for up to 48 hours to account for slower developmental times. Any hatched larvae were then transferred with the yeast paste into a fresh tube containing cornmeal medium. For \emph{D. simulans, D. yakuba} and \emph{D. pseudoobscura}, surviving adults were backcrossed to males or virgin females from the parental, wild-type strain. F1 progeny were then scored for eye-specific GFP expression, and transgenic lines were set up by crossing GFP-positive siblings. Because I was unable to identify flies carrying two copies of the transgene, these lines consisted of a mixed population of homozygous and heterozygous flies, meaning that the populations had to be periodically checked and GFP-positive flies selected in order to prevent loss of the transgene through genetic drift. For \emph{D. melanogaster}, surviving adults were backcrossed to \emph{w; Sco/SM6a} males or virgin females. F1 males were scored for eye-specific GFP expression and crossed singly to \emph{w; Sco/SM6a} virgins, then the same males were crossed to \emph{w; TM2/TM6c} virgins. F2 progeny of the \emph{Sco/SM6a} cross were scored for eye-specific GFP expression and a curly wing phenotype, while F2 progeny of the \emph{TM2/TM6c} cross were scored for eye-specific GFP expression and a Stubble phenotype. Siblings of each class were mated together. Balanced transgenic lines were identified in the F3 generation as stocks where all flies showed eye-specific GFP expression. 

\section{Immunohistochemistry}
Embryos were collected from each species after an overnight lay following the protocol described above. They were then dechorionated in 50\% bleach for 3 minutes, rinsed in cold water, and fixed by shaking for 20 minutes in 1.8 ml fixation solution (0.1 M PIPES, 1 mM MgSO4, 2 mM EGTA, pH 6.9) with 0.5 ml formaldehyde and 4 ml heptane. The aqueous phase was removed and 6 ml of methanol was added, followed by vortexing for 30 seconds. Any embryos that sank to the bottom of the tube were collected, rinsed with methanol, and stored at -20°C until needed for staining. Staining was performed as described \citep{patel_imaging_1994} with primary antibodies at the following concentrations: rabbit anti-Dichaete, 1:100; rabbit anti-SoxN, 1:100 or 1:50. Primary antibodies were detected with biotin-conjugated secondary antibodies (goat anti-rabbit) at 1:200 using the ABC Elite kit (Vectastain). Stained embryos were mounted in 70\% glycerol and photographed using Openlab v.4.0.2 imaging software on a Zeiss Axioplan microscope with a 20x objective.

\section{Chromatin immunoprecipitation}
For chromatin immunoprecipitations (ChIP), embryos were collected after an overnight or 12-hour lay and dechorionated as described above. They were fixed by shaking for 20 minutes in 670 μl crosslinking solution (50 mM HEPES, 1mM EDTA, 0.5 mM EGTA, 100 mM NaCl, pH 8.0) with 33 μl 37\% formaldehyde and 3 ml heptane added. The crosslinking reaction was stopped by centrifuging for 2 minutes at 1000g to pellet the embryos, removing the supernatant and adding 2 ml PBT with 125 mM glycine. Embryos were then weighed in an Eppendorf tube, flash-frozen in liquid nitrogen and stored at -80°C. Approximately 200 mg of embryos were used per biological replicate. ChIPs were performed as described with some modifications for a small amount of starting material \citep{ghavi-helm_analyzing_2012, sandmann_chip--chip_2007}. Embryos were homogenized in Eppendorf tubes using a plastic pestle rather than in a Dounce homogenizer. Each sample was homogenized for 30 seconds in 1 ml cold PBT supplemented with protease inhibitors (Complete Mini Protease Inhibitor cocktail tablets, Roche), then allowed to rest on ice for 30 seconds, then homogenized again for 30 seconds. The lysate was spun at 400g for 1 minute at 4°C, and the supernatant decanted into a fresh Eppendorf tube. After centrifugation at 1100g for 10 minutes at 4°C, the supernatant was discarded and the pellet resuspended in 1 ml cold cell lysis buffer supplemented with protease inhibitors. The sample was homogenized again for 30 seconds with a plastic pestle and the lysate spun at 2000g for 4 minutes at 4°C to pellet the nuclei. The pellet was resuspended in 1 ml cold nuclear lysis buffer and incubated for 20 minutes at room temperature to lyse the nuclei. 
\paragraph{}
A Diagenode Bioruptor was used for sonication, with the energy settings on high. Chromatin was sonicated in 100 μl aliquots for 16 cycles of 30 seconds on, 30 seconds off. A 50 μl input aliquot was removed from each sample and treated with RNaseA for 30 minutes at 37 °C, then with proteinase K overnight at 37 °C. Crosslinks were reversed by incubating at 65°C for 6 hours. The distribution of DNA fragment sizes was assessed by performing a phenol-chloroform extraction and running the resulting DNA on a 3\% agarose gel. Fragment sizes ranged from approximately 100 bp to 1000 bp, with the majority of the fragments falling between 300 and 700 bp. Immunoprecipitation was carried out with protein A-agarose beads (Millipore), using the buffers and wash protocol described in Sandmann et al. (2006). Anti-Dichaete antibody was pre-cleared by incubating for 3 hours at 4°C with methanol-fixed embryos, then added to a final concentration of 1:300. Affinity-purified anti-SoxN antibody was added without pre-clearing to a final concentration of 1:100. For mock IP controls, a rabbit anti-beta galactosidase antibody (AbCam) was added to a final concentration of 1:1000. After performing the immunoprecipitation, crosslinks were reversed and the DNA purified using the same protocol described above for input samples.
\subsection{ChIP-PCR}
Targets were chosen for PCR amplification to test the specific enrichment of each ChIP by examining previous ChIP-chip and DamID experiments carried out in our lab \citep{aleksic_role_2013,ferrero_soxneuro_2014}. For each gene, a highly bound interval was identified in \emph{D. melanogaster} and its sequence was used as a query to search the genome of each other species using BlastN \citep{altschul_basic_1990}, with the goal of identifying an orthologous region of 500-800 bp. A negative control region was also identified for each factor in each species where binding was not observed in previous experiments in \emph{D. melanogaster}. Primers were designed to amplify each region using Primer3 Plus \citep{untergasser_primer3plus_2007}. Oligonucleotide sequences are shown in Table 2.1. PCR conditions were identical for each set of samples and were as follows: 95°C for 2 min.; 45 cycles of 95°C for 30 sec., 58°C for 30 sec., 72°C for 30 sec.; 72°C for 5 min. 1 μl of ChIP, mock IP, or input DNA was used as a template for each reaction. PCR products were run out on a 1\% agarose gel, and the specificity of each antibody was assayed by comparing the presence and brightness of bands for the ChIP samples versus the mock IP and input samples.
\subsection{ChIP-chip}
Dichaete ChIP samples from \emph{D. melanogaster} were hybridized to a dual-color Nimblegen HD2 (2.1M probe) whole-genome tiling array in order to validate the specificity of the immunoprecipitation reactions. Probe libraries were constructed as described \citep{sandmann_chip--chip_2007}. ChIP samples and their respective mock IP controls were labelled with either Cy3 or Cy5 dyes, with a dye swap in one of three biological replicates. Each ChIP sample and its matched control were hybridized to the same microarray. Hybridization was performed according to the manufacturer’s specifications. Spot-finding was carried out using NimbleScan, a proprietary software package developed by Roche. The raw data were quantile-normalized in R and analyzed with two different peak-calling algorithms, TiMAT (http://bdtnp.lbl.gov/TiMAT/) and Ringo \citep{toedling_ringor/bioconductor_2007} at FDR values of 1\%, 5\%, 10\% and 25\%.
\subsection{ChIP-seq}
ChIP reactions for ChIP-sequencing were performed as described above, with the exception that the protein A-agarose beads were changed to protein A/G PLUS-agarose beads (Santa Cruz Biotechnology), as these do not contain salmon sperm DNA, a potential sequencing contaminant. Before library construction, sample concentrations were measured on a Qubit using the DNA High Sensitivity Assay (Life Technologies). Initially, libraries were constructed for sequencing in-house on an Ion Torrent PGM using the Ion Plus Fragment Library Kit (Life Technologies), quantified via qPCR, and templated using the Ion OneTouch Template Kit (Life Technologies). Libraries were sequenced on 316 chips; however, the quality and coverage of the resulting reads was insufficient for identifying binding peaks.
\paragraph{}
For the first attempt at Illumina sequencing, libraries were prepared using 10 ng of ChIP DNA or mock IP DNA, or the entire sample if less than 10 ng were available, with the NEBNext ChIP-Seq Library Prep Master Mix Set for Illumina (NEB). Samples were barcoded using the NEBNext Multiplex Oligos for Illumina (Index Primers 1-12) (NEB). For the second attempt, libraries were constructed using 10 ng of ChIP or input DNA, or the entire sample if less than 10 ng were available, with the TruSeq DNA LT Sample Prep kit (Illumina). Samples were barcoded using the indexed adapters included in the kit, which were diluted 1:250 to account for the low amount of starting material. Size selection was performed using Agencourt AMPure XP beads (Beckman Coulter), with the aim of recovering fragments between 250 and 400 bp. In all cases, the library concentrations were measured using a Qubit with the DNA High Sensitivity Assay (Life Technologies), and the size distributions of DNA fragments were measured using a 2100 Bioanalyzer with the High Sensitivity DNA kit and chips (Agilent). 10 nM libraries were sent to the EMBL GeneCore sequencing facility in Heidelberg, Germany (http://genecore3.genecore.embl.de/genecore3/index.cfm) for sequencing on an Illumina HiSeq 2000 with v3 chemistry, with 12 samples multiplexed per lane. All libraries were sequenced as 50-bp single-end reads.

\section{DamID}
\subsection{Cloning}
Three constructs were created for DamID: Dichaete-Dam, SoxN-Dam and Dam-only. The Dam coding sequence and the SoxN-Dam fusion protein coding sequence were initially cloned from existing pUAST vectors (Dam from Tony Southall, SoxN-Dam from Enrico Ferrero). The Dichaete-Dam fusion protein coding sequence was cloned from genomic DNA extracted from a \emph{D. melanogaster} line carrying this construct, which was created by Faysal Riaz (cite thesis). For Dam and SoxN-Dam, primers were designed to amplify the coding regions of Dam or the SoxN-Dam fusion protein as well as upstream UAS sites, an \emph{HSP70} promoter, and the \emph{SV40} 5’ UTR. For Dichaete-Dam, primers were designed to amplify the fusion protein coding region, upstream UAS sites, an \emph{HSP70} promoter, and the \emph{kayak} 5’ UTR. All primer sequences are available in Table 2.2. A forward primer was designed to introduce a \emph{Spe}I site upstream of the cloned region for Dichaete-Dam and SoxN-Dam, and a reverse primer introduced an \emph{Avr}II site downstream of the Dichaete-Dam cloned region.
\paragraph{}
A two-step cloning process was employed, first cloning inserts into the pSLfa1180fa shuttle vector, and then from the shuttle vector into a \emph{piggyBac} vector marked with 3xP3-EGFP, pBac{3xP3-EGFPafm} \citep{horn_versatile_2000}. To clone all inserts into the shuttle vector, PCR amplicons and the pSLfa1180fa vector were cut with the following restriction enzymes (NEB):
\begin{itemize}
	\item SoxN-Dam, pSLfa1180fa: \emph{Spe}I/\emph{Stu}I
	\item Dichaete-Dam, pSLfa1180fa: \emph{Spe}I/\emph{Avr}II
	\item Dam, pSLfa1180fa: \emph{Sph}I/\emph{Stu}I
\end{itemize}
    

