\chapter{Exploratory Analysis of Dichaete and SoxNeuro in Four Species of \emph{Drosophila}}

\hrulefill

\section{Overview and motivation}

Before performing genome-wide binding experiments for Dichaete and SoxN in non-model species of \emph{Drosophila}, I set out to characterize the orthologous proteins in each species of interest at the levels of sequence, expression pattern and specific target binding. In addition to verifying that the orthologous transcription factors are similar enough to make direct inter-species comparisons of binding valid, this process also allowed me to test the specificity of the Dichaete and SoxN antibodies that I planned to use for ChIP-seq in each species. Although the HMG DNA-binding domains of group B Sox proteins are highly conserved over the evolutionary distances that I examined, other domains of Dichaete and SoxN have diverged somewhat, making it possible that antibodies raised against these proteins in one species might not display the same reactivity with targets in another species \citep{mckimmie_conserved_2005}. I analyzed the expression pattern of each protein by collecting embryos from each species and performing immunohistochemistry using both Dichaete and SoxN antibodies. I then compared the resulting staining patterns at a variety of developmental stages.\\

Because antibodies can often work well in immunohistochemistry but be ineffective in ChIP experiments, it was also necessary to test for enrichment of specific target sequences using ChIP-PCR before proceeding to ChIP-seq. This was an imprecise process in non-model species because PCR targets had to be designed in each species based on regions of known binding, which were only available in \emph{D. melanogaster}. Although orthologous sequences can be approximated using tools such as BLAST \citep{altschul_basic_1990} or by simply examining gene models and identifying, for example, introns at orthologous positions in each species, it is unknown \emph{a priori} whether the transcription factors of interest will bind to those exact sequences in each species. Nonetheless, I designed primers for three potential target regions for each transcription factor in each species, along with one negative control region for each transcription factor, which were chosen by examining the available \emph{in vivo} binding data in \emph{D. melanogaster} and identifying regions where no binding was observed. I performed each ChIP reaction on three biological replicates of chromatin derived from fixed embryos collected from each species, using both input chromatin and a mock IP as controls. I then tested for target enrichment by performing PCR using each of the primer sets that I designed, with the ChIP DNA, the mock IP DNA, and the input as templates. Although qPCR would have given a more quantitative view of the enrichment of target sequences in the ChIP DNA compared to the control, classical PCR yielded a qualitative overview of the presence or absence of target sequences in each sample.\\

In order to compare Dichaete and SoxNeuro binding patterns on a genome-wide scale between four \emph{Drosophila} species, I initially intended to use ChIP-seq. As discussed in the introduction, this technique has both advantages and disadvantages; ChIP allows for the detection of binding by the endogenous protein in the native spatial and temporal context, provided a suitable antibody is available. Genome-wide binding can be measured from ChIP experiments either by hybridizing the resulting DNA libraries to a microarray (ChIP-chip) or sequencing them on a high-throughput sequencer (ChIP-seq). I chose to primarily use sequencing rather than microarrays because of the lack of availability of tiling arrays for non-model \emph{Drosophila} species and also because of the increase in resolution and dynamic range possible with sequencing in comparison to microarray technology \citep{aleksic_chiping_2009}. However, I also performed one ChIP-chip experiment in \emph{D. melanogaster} in order to further validate the Dichaete antibody before investing in sequencing.\\

For both the ChIP-chip and ChIP-seq experiments, I planned to sequence 3 replicates of each experimental condition (Dichaete and SoxN ChIP) and 3 control replicates in each species studied. I planned to compare the same control replicates against each experimental condition; however, due to the lack of target enrichment in ChIP-PCR experiments with the SoxN antibody, I only performed ChIP-chip and ChIP-seq for Dichaete. Although high-quality ChIP-seq biological replicates often show high correlation with each other, leading the modENCODE Consortium to accept a minimum of 2 biological replicates per experiment \citep{landt_chip-seq_2012}, I decided to use 3 replicates to provide greater statistical confidence and in order to reduce the effect of any potential technical problems that could lead to one replicate being an outlier. \\

Theoretically, the most appropriate control for a ChIP experiment is a mock IP, in which preimmune serum or an antibody that is not expected to bind to anything in the sample is used in parallel to the real IP; this type of control is commonly used in ChIP-chip experiments \citep{ghavi-helm_analyzing_2012, park_widespread_2013}. I used a mock IP control with an antibody against beta-Galactosidase for the Dichaete ChIP-chip experiment and for one Dichaete ChIP-seq experiment. However, mock IPs often yield very low amounts of DNA, making it difficult to construct sequencing libraries and leading to the introduction of bias from PCR overamplification. For this reason, input chromatin, which is a sample of the original chromatin that is set aside after sonication and then purified without performing any type of immunoprecipitation, is often used instead of a mock IP for ChIP-seq \citep{ghavi-helm_analyzing_2012}. An input control helps to correct for some of the biases associated with high-throughput sequencing, which can stem from factors such as GC content, differences in mappability and local chromatin accessibility, because it is subject to the same sources of bias as the experimental sample \citep{dohm_substantial_2008}. After observing high amounts of PCR overamplification in my mock IP control samples for ChIP-seq, I switched to using input controls for all subsequent ChIP-seq experiments. In total, I generated the following genome-wide ChIP datasets: three replicates of Dichaete ChIP-chip with three replicates of beta-Galactosidase mock IP controls in \emph{D. melanogaster}; four, two and three replicates respectively of Dichaete ChIP-seq with beta-Galactosidase mock IP controls in \emph{D. melanogaster, D. simulans} and \emph{D. yakuba}; and three replicates of Dichaete ChIP-seq with three replicates of input chromatin controls in \emph{D. melanogaster, D. simulans, D. yakuba and D. pseudoobscura}. A detailed description of methods used for both immunohistochemistry and all ChIP-based techniques can be found in Chapter 2.

\section{Sequence and phylogenetic analysis}
Previous to the work of this thesis, the amino acid sequences of the group B Sox proteins Dichaete, SoxN, Sox21a and Sox21b had been aligned in the insects \emph{Drosophila melanogaster}, \emph{Drosophila pseudoobscura}, \emph{Anopheles gambiae} and \emph{Apis mellifera}, along with the corresponding orthologous sequences in mouse, revealing a deep conservation of the HMG box DNA binding domains in each protein alongside considerably higher divergence in other domains of the proteins. Additionally, the amino acid sequences of just the HMG domains of all known group B Sox proteins at the time were aligned together, illustrating the high levels of sequence conservation among all group B Sox proteins as well as specific amino acid substitutions that are common among orthologs but differ between paralogous proteins in each species analyzed \citep{mckimmie_conserved_2005}. On a larger scale, the genomic organization of group B \emph{Sox} genes, with \emph{Dichaete, Sox21a} and \emph{Sox21b} located nearby on the same chromosome in the “Dichaete cluster” and \emph{SoxN} located on a separate chromosome, has been shown to be conserved in insects ranging from \emph{D. melanogaster} to the hymenopteran \emph{Nasonia vitripennis} and the coleopteran \emph{Tribolium castaneum}. An independent duplication in \emph{Tribolium} has resulted in a fifth group B \emph{Sox} gene, also located in the \emph{Dichaete} cluster \citep{phochanukul_no_2010}. Previous phylogenetic analysis of group B Sox proteins in insects has shown that orthologs for each family member cluster into clades together, supporting the hypothesis that the four common group B Sox proteins diverged before the radiation of the major insect phyla \citep{wilson_evolution_2008,zhong_parallel_2011}.\\

I extended this analysis and focused it on the species relevant to my study by using BLAST to identify the orthologous sequences of the group B Sox proteins in \emph{D. melanogaster, D. simulans, D. yakuba} and \emph{D. pseudoobscura} \citep{altschul_basic_1990}. I used ClustalW2 to align the entire amino acid sequences of each orthologous protein in the four species and found that they are very highly conserved, with near perfect conservation in the HMG domains and a relatively low number of substitutions and indels in other areas of the proteins (Figure 3.1A-B) \citep{chenna_multiple_2003}. The most divergent sequences belong to \emph{D. pseudoobscura}, which is the farthest from the other three species phylogenetically. In \emph{D. simulans}, I observed a large deletion at the N-terminal end of SoxNeuro; however, it was uncertain whether this was a true deletion or a fragment of missing sequence due to the lower quality of the \emph{D. simulans} genome assembly. I also used the HMG domains from these species along with the amino acid sequences from the HMG domains of all identified group B Sox proteins in the mosquito \emph{Aedes aegypti}, the beetle \emph{Tribolium castaneum}, the honeybee \emph{Apis mellifera}, the nematode \emph{C. elegans} and the vertebrates mouse (\emph{Mus musculus}) and human (\emph{Homo sapiens}) to construct a neighbor-joining tree (Figure 3.1C). I used an established outgroup, the fungal protein MATA-1, to root the tree \citep{laudet_ancestry_1993}. This analysis shows that, for each group B Sox protein, the orthologous sequences from the four \emph{Drosophila} species form a monophyletic clade, with the nearest sister group in each case except for that of Dichaete being the orthologous protein in \emph{A. aegypti}. The results of the phylogenetic analysis of group B Sox proteins support the idea that orthologous proteins in \emph{Drosophila} are highly conserved and are suitable for an inter-species comparison of binding. Nonetheless, I still needed to test whether antibodies against Dichaete and SoxN in \emph{D. melanogaster} would react specifically with the orthologous proteins in each other species; to do so, I proceeded to use immunohistochemistry and ChIP-PCR.

\begin{figure}[H]
\centering
\includegraphics{fig3-1a}
\label{Figure 3.1}
\end{figure}

\begin{figure}[H]
\centering
\includegraphics{fig3-1c}
\caption{Phylogenetic analysis of group B Sox amino acid sequences. A.) Multiple alignment of entire amino acid sequence of Dichaete in, starting from the top, \emph{D. melanogaster, D. simulans, D. yakuba} and \emph{D. pseudoobscura}. The HMG domains of each orthologous protein are highlighted in the red box and are nearly identical. B.) Multiple alignments of entire amino acid sequence of SoxNeuro in, starting from the top, \emph{D. melanogaster, D. simulans, D. yakuba} and \emph{D. pseudoobscura}. The HMG domains of each orthologous protein are highlighted in the red box and are nearly identical. C.) Multiple alignment of entire Neighbor-joining tree constructed from multiple alignment of the amino acid sequences of group B Sox HMG domains from the four species of \emph{Drosophila} of interest as well as several other invertebrates and vertebrates. Species used in this study are highlighted in blue (\emph{D. melanogaster}), red (\emph{D. simulans}), green (\emph{D. yakuba}) and purple (\emph{D. pseudoobscura}). The tree was rooted using the fungal protein MATA-1, an established outgroup \citep{laudet_ancestry_1993}. The sequences from orthologous proteins in each species of \emph{Drosophila} form monophyletic clades, with the nearest outgroup in each case being \emph{A. aegypti}. Abbreviations: \emph{Drosophila melanogaster (Dmel), Drosophila simulans (Dsim), Drosophila yakuba (Dyak), Drosophila pseudoobscura (Dpse), Aedes aegypti (Aaeg), Tribolium castaneum (Tcas), Apis mellifera (Amel), Caenorhabditis elegans (Cele), Homo sapiens (Hsap), Mus musculus (Mmus)}.}
\label{Figure 3.1}
\end{figure}

\section{Assessing expression patterns}
The specific gene regulatory activity of transcription factors is tightly coupled to their spatial expression patterns, which often change throughout development. In \emph{Drosophila}, even orthologous \emph{cis}-regulatory regions with divergent sequences and positioning of TF binding sites have been shown to drive equivalent patterns of expression in transgenic assays \citep{hare_sepsid_2008}; it has been speculated that this phenotypic conservation is due to the evolution of compensatory binding events. Binding of TFs to shadow enhancers, which are secondary regions of regulatory DNA often located farther away from their target genes than primary enhancers and which can drive nearly identical expression patterns, can also confer robustness on expression \citep{ludwig_consequences_2011,perry_shadow_2010}. However, there are also well-documented cases in which evolutionary changes in the \emph{cis}-regulatory region of a transcription factor have resulted in both a change in its expression pattern and the regulation of its downstream target genes, yielding novel phenotypes \citep{arnoult_emergence_2013,frankel_conserved_2012}. Therefore, before examining the genome-wide binding patterns of Dichaete and SoxN in different species of \emph{Drosophila}, I first wanted to examine the expression patterns of each orthologous protein in each species of interest in order to verify that they were expressed in grossly equivalent domains during each stage of development. In order to do so, I performed immunohistochemistry on embryos collected from each species using antibodies for Dichaete and SoxN raised against the \emph{D. melanogaster} proteins \citep{ferrero_soxneuro_2014,soriano_drosophila_1998}; this also served the purpose of determining whether the antibodies would react specifically with their respective orthologous proteins in each species.\\

The expression patterns of Dichaete and SoxN in the \emph{D. melanogaster} embryo have been previously characterized using immunohistochemistry, fluorescent immunohistochemistry and \emph{in situ} hybridizations, as well as in the \emph{D. pseudoobscura} embryo using \emph{in situ} hybridizations \citep{cremazy_<_2000,mckimmie_conserved_2005,overton_evidence_2002,soriano_drosophila_1998}. Using these as references for comparison, I stained whole embryos from \emph{D. melanogaster, D. simulans, D. yakuba} and \emph{D. pseudoobscura} for Dichaete and SoxN and examined the expression patterns of Dichaete and SoxN at different stages of embryonic development (Figure 3.2, Figure 3.3). I observed stronger staining with the Dichaete antibody than the SoxN antibody in all species; nonetheless, a clear and specific pattern of staining could be observed for both proteins in embryos of all species. Qualitatively, the spatial and temporal expression patterns of both proteins in all species were extremely similar. For Dichaete, a broad domain of staining was clearly visible in the blastoderm at stage 5, with a smaller anterior stripe. Characteristic strong staining in the central nervous system appeared at stage 9, with expression detectable in the neuroectoderm and ventral midline in stages 9 and 11. At stage 13, staining became visible in the hindgut, and ectodermal stripes appeared at stage 16. For SoxN, staining was visible throughout the neuroectoderm at stage 8, but was excluded from the ventral midline. The neuroectodermal expression began to take on a segmental pattern at stage 10. The segmental stripes were extended laterally at stage 12, and at stage 16 ectodermal stripes were apparent. These observations provide evidence that both Dichaete and SoxN are expressed in equivalent spatial and temporal patterns during embryonic development in \emph{D. melanogaster, D. simulans, D. yakuba} and \emph{D. pseudoobscura}.

\begin{figure}
\centering
\includegraphics{fig3-2}
\caption{Dichaete expression patterns in developing embryos from \emph{D. melanogaster, D. simulans, D. yakuba} and \emph{D. pseudoobscura}. \emph{D. melanogaster} stainings are reproduced from \citet{soriano_drosophila_1998} and were taken at stages 9, 11, 13 and 16. For all other species, images were taken at stages 5, 9, 11, 13 and 16. A.) Lateral views of Dichaete expression in \emph{D. melanogaster} embryos. Black arrows indicate the brain and the white arrow indicates the hindgut. Black arrowheads indicate the chordotonal organs. B-D, lateral views; B'-D', dorsal or ventral views. B.) and B'.) Dichaete expression in \emph{D. simulans} embryos. C.) and C'.) Dichaete expression in \emph{D. yakuba} embryos. D.) and D'.) Dichaete expression in \emph{D. pseudoobscura} embryos. Expression patterns are qualitatively the same at the equivalent stages in each species. White arrows in B', C' and D' indicate staining in the ventral midline.}
\label{Figure 3.2}
\end{figure}

\begin{figure}
\centering
\includegraphics{fig3-3}
\caption{SoxNeuro expression patterns in developing embryos from \emph{D. melanogaster, D. simulans, D. yakuba} and \emph{D. pseudoobscura}. \emph{D. melanogaster} stainings are reproduced from \citet{buescher_formation_2002}. Images were taken at stages 8, 10, 12 and 16. A.) Ventral views of SoxN expression in \emph{D. melanogaster} embryos. B-D, lateral views; B'-D', dorsal or ventral views. B.) and B'.) SoxN expression in \emph{D. simulans} embryos. C.) and C'.) SoxN expression in \emph{D. yakuba} embryos. D.) and D'.) SoxN expression in \emph{D. pseudoobscura} embryos. Although staining is weaker than for Dichaete, expression patterns are qualitatively the same at the equivalent stages in each species. White arrows in B' and C' indicate lack of expression in the ventral midline at early stages.}
\label{Figure 3.3}
\end{figure}

\section{Targeted binding analysis}
After determining that Dichaete and SoxN have comparable patterns of expression in each species of interest and that the antibodies against each protein were capable of reacting specifically with the orthologous protein in each species, I decided to test the efficacy of the antibodies in chromatin immunoprecipitation (ChIP), as not all antibodies that work well for immunohistochemistry also work well in ChIP reactions \citep{landt_chip-seq_2012}. To do so, I performed ChIP-PCR using chromatin derived from embryos of each species. This also served to determine whether enrichment for specific known targets of Dichaete and SoxN could be detected in non-\emph{melanogaster} species. Although both the Dichaete and SoxN antibodies have been previously used in ChIP-chip experiments, in both cases the data was of variable quality, making it worthwhile to test the antibodies with a targeted analysis before proceeding to perform ChIP-seq (Aleksic, 2011; Ferrero, 2014a).\\

I performed ChIP on three biological replicates of chromatin from each species with each antibody. For each replicate, I also performed a mock IP control and set aside an aliquot of input chromatin, resulting in nine samples per TF per species. For each TF, I identified three high-confidence target intervals in \emph{D. melanogaster} as well as one control region where binding was not detectable \citep{aleksic_role_2013,ferrero_soxneuro_2014}. I then used PCR to amplify the target regions as well as the control region in each sample, testing the enrichment of the ChIP samples in comparison to both the input and the mock IP control samples. The target regions chosen for Dichaete were in regulatory regions of the genes \emph{slit} (\emph{sli}), \emph{achaete} (\emph{ac}) and \emph{commissureless} (\emph{comm}), and the negative control region was near the gene \emph{klingon} (\emph{klg}). For the \emph{D. melanogaster} Dichaete ChIP samples, strong amplification was visible at each target locus, whereas little or no amplification was visible for the mock IP control samples (Figure 3.4). As expected, the input samples also showed amplification at each target region; however, this was not as bright as the amplification present in the ChIP samples. Unexpectedly, amplification was also present at the negative control region for both the ChIP samples and the input samples. As ChIP is an inherently noisy technique, it is difficult to know whether this was due to true, previously undetected binding of Dichaete in this region or to contamination of the ChIP samples by non-specifically bound chromatin \citep{aleksic_chiping_2009,buck_chip-chip:_2004}.\\

\begin{figure}
\centering
\includegraphics{fig3-4}
\caption{ChIP-PCR for Dichaete targets in \emph{D. melanogaster}. Lanes 1-9, PCR for \emph{slit} target region. Lanes 11-19, PCR for \emph{achaete} target region. Lane 10, negative control. Lanes 1-3 and 11-13 are three replicates from Dichaete ChIP DNA, lanes 4-6 and 14-16 are three replicates from mock IP control DNA, and lanes 7-9 and 17-19 are three replicates from input chromatin, which was set aside from each sample before immunoprecipitation. Amplification is strongly visible in all Dichaete ChIP samples for both targets, indicating enrichment of target sequences, while little or no amplification is visible for any controls. As expected, amplification is also visible for input chromatin.}
\label{Figure 3.4}
\end{figure}

A similar pattern of enrichment was observed in each other species, with some variation in the strength of amplification at different target regions. In the \emph{D. simulans} ChIP samples, the \emph{sli} and \emph{comm} regions were strongly amplified, while the \emph{ac} region was much weaker, as was the \emph{klg} negative control region. In the \emph{D. pseudoobscura} ChIP samples, the \emph{sli} and \emph{ac} regions were strongly amplified, while the \emph{comm} region showed no amplification. In the \emph{D. yakuba} ChIP samples, amplification was present in all target regions, but at a lower level than for the other species and also at a lower level than the corresponding input samples. However, given that the PCR probes were designed based solely on sequence orthology, without any direct evidence of \emph{in vivo} binding in species other than \emph{D. melanogaster}, this variation was not surprising.\\

For SoxN, the targets chosen were in regulatory regions of the genes \emph{nervous fingers 1} (\emph{nerfin-1}), \emph{glial cells missing-2} (\emph{gcm-2}) and \emph{castor} (\emph{cas}), and the negative control region was near the gene \emph{Protein phosphatase D6} (\emph{PpD6}). However, unlike the Dichaete samples, the SoxN ChIP samples did not show a pattern of significant target enrichment in PCR assays. While amplification of each target region was generally detectable for the input DNA in all species, it was weak and variable in both the ChIP samples and mock IP negative controls. To determine whether this lack of enrichment was due to an inappropriate selection of target loci, I performed PCRs on the same samples with a new set of primers designed to detect regions that had been identified as SoxN targets in an earlier study \citep{girard_chromatin_2006} (primers from E. Ferrero). Again, no significant enrichment was observed in experimental samples versus negative controls. For all replicates, less DNA was recovered from the IP reaction for SoxN than for Dichaete, raising the question of whether the lack of target amplification in the SoxN ChIP samples was due to lack of specificity of the antibody or simply an insufficient quantity of template DNA. However, given the results of all of the ChIP-PCR experiments, I decided to focus primarily on Dichaete for performing ChIP-seq.

\section{Genome-wide binding analysis of Dichaete via ChIP-chip and ChIP-seq}
\subsection{ChIP-chip for Dichaete in \emph{D. melanogaster}}
Initially, I used three biological replicates to perform a ChIP-chip experiment for Dichaete in \emph{D. melanogaster} in order to further validate the ChIP-PCR results. Of these three, one produced highly noisy data, and the remaining two suffered from problems during loading of the microarrays. These two replicate ChIP/control pairs were hybridized to new microarrays; however, one of the new arrays leaked during hybridization and had to be discarded. The best resulting ChIP/control pair was therefore analyzed on its own, using the software tools TiMAT (\url{http://bdtnp.lbl.gov/TiMAT/}) and Ringo \citep{toedling_ringo_2007}. TiMAT found 5444 peaks at FDR1, 9807 peaks at FDR5, 12822 peaks at FDR10, and 19044 peaks at FDR25 for this dataset, while Ringo found 10322 peaks at FDR1, 18724 peaks at FDR5, 23189 peaks at FDR10, and 31915 peaks at FDR25. The TiMAT FDR1 results were chosen for further analysis, as they represent the most stringent and high-confidence dataset, and are also in line with the number of peaks predicted by previous ChIP-chip and DamID experiments for Dichaete \citep{aleksic_role_2013}.\\

Each interval in the TiMAT FDR1 dataset was assigned to the closest gene within 10kb upstream or downstream, with intervals that fell an equal distance between two genes being assigned to both. This resulted in 3807 gene assignments. The list of genes was uploaded onto FlyMine (\url{www.flymine.org}) \citep{lyne_flymine:_2007}, and the Gene Ontology Enrichment widget was used with a Holm-Bonferroni correction for multiple hypothesis testing to determine the what terms from the biological process ontology were enriched in the putative target genes. Terms with the most significant p-values included organ development (p = 1.19E-43), anatomical structure morphogenesis (p = 5.81E-42), biological regulation (p = 6.14E-37), generation of neurons (p = 9.06E-34) and neuron differentiation (p = 1.82E-30). A graphical overview of biological process enrichments was created using the Ontologizer and the PANTHER GOSlim ontology, which is a subset of the Gene Ontology containing high-level terms (Figure 3.5) \citep{bauer_ontologizer_2008}. When the TiMAT FDR1 binding intervals were visualized against the genome using IGB \citep{nicol_integrated_2009} binding was observed at known Dichaete targets such as \emph{slit} and \emph{commissureless} (Figure 3.6).\\

\begin{figure}
\centering
\includegraphics{fig3-5_17cm}
\caption{Gene Ontology Biological Process GOSlim terms enriched in annotated targets of Dichaete ChIP-chip binding intervals in \emph{D. melanogaster}. All terms highlighted in green are statistically significant (p \textless 0.05) after correction for multiple hypothesis testing, with the intensity of the green correlating to lower p-values. Arrows go from child terms in the ontology to parent terms, which are related by either an “is\_a” or a “part\_of” relationship.}
\label{Figure 3.5}
\end{figure}

A crude comparison to previous Dichaete binding datasets was performed by taking the intersection of the list of FDR1 gene hits with a list of core Dichaete target genes compiled by J. Aleksic from three ChIP-chip datasets and one DamID dataset, using the intersect tool on FlyMine \citep{aleksic_role_2013,lyne_flymine:_2007}. This intersection contains 1626 genes, representing 43\% of the FDR1 gene list and 34\% of the core target gene list. An intersection of the list of FDR5 gene hits with the same core target gene list contains 2330 genes, representing 40\% of the FDR5 gene list and 48\% of the core target gene list. These percentages are within the range of expected values based on pairwise comparisons of previously generated Dichaete binding datasets. Although the data were somewhat noisy and I was only able to use one biological replicate, overall the ChIP-chip results supported the hypothesis that the Dichaete antibody was specifically binding to Dichaete protein and pulling down true target sequences.

\begin{figure}
\centering
\includegraphics{fig3-6}
\caption{Dichaete ChIP-chip binding at known Dichaete targets in \emph{D. melanogaster}. Gene models are in black, the Dichaete ChIP-chip binding profile is in blue and FDR1 Dichaete binding intervals are represented by blue bars above the binding profile. The Dichaete binding profile represents log2 ratio scores of Dichaete ChIP intensity versus mock IP intensity at each probe on the microarray. A.) Dichaete binding in several introns of the gene \emph{slit}, on chromosome 2R. B.) Dichaete binding in the intron and downstream of the gene \emph{comm}, on chromosome 3L.}
\label{Figure 3.6}
\end{figure}

\subsection{ChIP-seq for Dichaete in four species of \emph{Drosophila}}
\subsubsection{Sequencing on the Ion Torrent PGM}
In a first attempt at performing ChIP-seq, three biological replicates of Dichaete ChIP samples from \emph{D. melanogaster}, along with three matched replicate mock IP control samples, were sequenced in-house on an Ion Torrent PGM. Difficulties in generating the correct enrichment of templated DNA molecules from each library on the Ion Sphere Particles (ISPs) and in loading the ISPs onto the sequencing chips led to highly variable and low numbers of reads from each run. A summary of the reads generated for each sample can be found in Table 3.1.
 
\begin{table}[h]
\centering
\begin{tabular}{|l|l|}
\hline
\textbf{Sample}          & \textbf{Raw reads} \\ \hline
Dichaete ChIP 1 & 64,204    \\ \hline
Dichaete ChIP 2 & 601,960   \\ \hline
Dichaete ChIP 3 & 2,354,296 \\ \hline
Mock IP 1       & 318,257   \\ \hline
Mock IP 2       & 1,210,355 \\ \hline
Mock IP 3       & 85563    \\ \hline
\end{tabular}
\caption{Summary of reads obtained for ChIP-seq libraries on the Ion Torrent PGM}
\label{Table 3.1}
\end{table}

Given the insufficient numbers of reads, as well as the high amount of variation in coverage between replicate samples and generally low quality scores obtained, these datasets were not analyzed further. In the hope of getting more reads per sample, I decided to move to Illumina sequencing.

\subsubsection{Sequencing on the Illumina HiSeq}

I attempted to perform ChIP-seq for Dichaete in embryos from all four species of \emph{Drosophila} on the Illumina HiSeq 2000 platform using two different controls: a mock IP and input chromatin. In the first instance, I generated and sequenced matched ChIP-seq and mock IP libraries for one biological replicate of \emph{D. melanogaster} chromatin, two biological replicates of \emph{D. simulans} chromatin and three biological replicates of \emph{D. yakuba} chromatin. All of these samples were multiplexed in one lane and sequenced as 50-bp single end reads. On average, Illumina sequencing resulted in many more reads and higher quality scores for each sample than Ion Torrent sequencing. However, I also observed a very high level of duplication in all samples (Table 3.2). Although some duplication may be expected in a ChIP-seq dataset with very strong enrichment of target sequences, for this dataset the majority of the duplicates were 10-fold or more, indicating that the most likely cause of duplication was PCR overamplification during the library preparation step \citep{bardet_computational_2011}. High duplication was also present in the mock IP control samples; theoretically, these samples should not display significant enrichment, meaning that the duplication was again likely due to PCR overamplification stemming from a very low amount of starting material. After mapping the reads to their respective genomes and visualizing the read densities, it was clear that most of the sequenced reads for the mock IP samples were PCR artefacts, as they formed discrete, high peaks, rather than a random background distribution (Figure 3.7). There was also evidence for contamination by adapter sequences, as a relatively low fraction of reads from each sample mapped uniquely to the genome. I attempted to call peaks for each ChIP-control pair using MACS; while MACS was able to identify some peaks in each sample, the numbers of peaks ranged widely (from 16 in the \emph{D. melanogaster} sample to 4006 in the \emph{D. yakuba} replicate 1), and in most cases more negative peaks were called than positive peaks.

\begin{table}[h]
\centering
\begin{tabular}{|l|l|l|l|}
\hline
\textbf{Sample}            & \textbf{Clean reads} & \textbf{Mapped reads} & \textbf{\% Duplicate reads} \\
\hline
\emph{D. mel} Dichaete 1 & 19,504,942  & 14,022,763   & 96.6               \\ \hline
\emph{D. mel} mock IP 1  & 19,334,291  & 9,887,936    & 97.0                 \\ \hline
\emph{D. sim} Dichaete 1 & 19,699,015  & 8,601,165    & 96.8               \\ \hline
\emph{D. sim} Dichaete 2 & 15,322,447  & 6,482,716    & 96.7               \\ \hline
\emph{D. sim} mock IP 1  & 19,567,960  & 9,130,168    & 96.4               \\ \hline
\emph{D. sim} mock IP 2  & 22,398,847  & 7,244,943    & 96.5               \\ \hline
\emph{D. yak} Dichaete 1 & 17,031,519  & 5,227,683    & 96.1               \\ \hline
\emph{D. yak} Dichaete 2 & 17,577,632  & 4,742,744    & 95.8               \\ \hline
\emph{D. yak} Dichaete 3 & 19,569,826  & 5,224,909    & 96.7               \\ \hline
\emph{D. yak} mock IP 1  & 8,562,845   & 5,391,420    & 96.1               \\ \hline
\emph{D. yak} mock IP 2  & 17,792,354  & 8,539,244    & 95.8               \\ \hline
\emph{D. yak} mock IP 3  & 20,003,005  & 13,004,641   & 97.2               \\ \hline
\end{tabular}
\caption{Summary of reads obtained for ChIP-seq libraries with mock IP controls on the Illumina HiSeq 2000. Abbreviations: \emph{D. mel, Drosophila melanogaster; D. sim, Drosophila simulans; D. yak, Drosophila yakuba}.}
\label{Table 3.2}
\end{table}

\begin{figure}
\centering
\includegraphics{fig3-7}
\caption{Dichaete ChIP-seq reads in \emph{D. melanogaster} with mock IP control. A 70-kb region around the gene \emph{slit} is shown. All samples are scaled to 1,000,000 reads for visualization purposes; the y-axis ranges from 0 to 40. Top track, Dichaete ChIP-seq mapped reads. Bottom track, mock IP control mapped reads. The mock IP reads show a sparser distribution with high, narrow peaks, indicative of PCR overamplification.}
\label{Figure 3.7}
\end{figure}

Given these results, I decided to repeat the ChIP experiments and to switch to using input chromatin as controls, as they should be less vulnerable to PCR overamplification. I also changed the strategy for constructing libraries, switching from the NEBNext library kit to the Illumina TruSeq kit, which comes with pre-barcoded adapters, in order to try to decrease adapter contamination. I generated and sequenced matched ChIP and input libraries for three biological replicates each of \emph{D. melanogaster, D. simulans, D. yakuba} and \emph{D. pseudoobscura} chromatin. The samples were multiplexed in two lanes and sequenced as 50-bp single end reads. For some samples, considerably less reads were generated than in the previous sequencing attempt, due to technical variation of the sequencer (Table 3.3). Overall, the rates of duplication improved; however, they were still relatively high, indicating that both adapter contamination and PCR overamplification continued to be problematic.

\begin{table}[h]
\centering
\begin{tabular}{|l|l|l|l|}
\hline
\textbf{Sample}            & \textbf{Clean reads} & \textbf{Mapped reads} & \textbf{\% Duplicate reads} \\ \hline
\emph{D. mel} Dichaete 1 & 16,476,257  & 3,039,886    & 92.2               \\ \hline
\emph{D. mel} Dichaete 2 & 3,884,815   & 869,958      & 89.9               \\ \hline
\emph{D. mel} Dichaete 3 & 15,694,790  & 5,680,372    & 92.4               \\ \hline
\emph{D. mel} input 1    & 565,635     & 178,253      & 94.4               \\ \hline
\emph{D. mel} input 2    & 6,873,349   & 3,687,833    & 73.7               \\ \hline
\emph{D. mel} input 3    & 12,277,355  & 6,619,627    & 71.2               \\ \hline
\emph{D. sim} Dichaete 1 & 6,053,616   & 1,796,028    & 88.6               \\ \hline
\emph{D. sim} Dichaete 2 & 6,391,812   & 1,074,217    & 86.7               \\ \hline
\emph{D. sim} Dichaete 3 & 542,281     & 252,277      & 71.4               \\ \hline
\emph{D. sim} input 1    & 1,778,512   & 798,846      & 91.0                 \\ \hline
\emph{D. sim} input 2    & 9,951,360   & 4,607,383    & 87.5               \\ \hline
\emph{D. sim} input 3    & 2,277,242   & 937,074      & 87.1               \\ \hline
\emph{D. yak} Dichaete 1 & 3,641,069   & 1,957,927    & 70.7               \\ \hline
\emph{D. yak} Dichaete 2 & 5,227,275   & 845,896      & 69.8               \\ \hline
\emph{D. yak} Dichaete 3 & 12,480,232  & 1,861,494    & 93.7               \\ \hline
\emph{D. yak} input 1    & 9,886,306   & 5,354,002    & 84.5               \\ \hline
\emph{D. yak} input 2    & 3,791,180   & 1,948,138    & 42.8               \\ \hline
\emph{D. yak} input 3    & 7,091,621   & 3,966,281    & 82.4               \\ \hline
\emph{D. pse} Dichaete 1 & 7,552,737   & 4,519,842    & 69.9               \\ \hline
\emph{D. pse} Dichaete 2 & 9,716,950   & 2,473,247    & 91.9               \\ \hline
\emph{D. pse} Dichaete 3 & 3,922,525   & 1,939,907    & 83.9               \\ \hline
\emph{D. pse} input 1    & 19,351,331  & 11,090,369   & 80.6               \\ \hline
\emph{D. pse} input 2    & 18,854,725  & 8,908,703    & 69.5               \\ \hline
\emph{D. pse} input 3    & 13,679,916  & 8,788,161    & 89.5               \\ \hline
\end{tabular}
\caption{Summary of reads obtained for ChIP-seq libraries with input controls on the Illumina HiSeq 2000. Abbreviations: \emph{D. mel, Drosophila melanogaster; D. sim, Drosophila simulans; D. yak, Drosophila yakuba; D. pse, Drosophila pseudoobscura}}
\label{Table 3.3}
\end{table}

In order to assess the level of reproducibility between biological replicates, I calculated the Pearson's correlation coefficient (PCC) between the read densities for each set of replicates for the same condition, using a script provided by \citet{bardet_computational_2011}. The highest PCCs between pairs of replicates ranged from 0.71-0.85; however, these values are still below the anticipated PCC of \textgreater 0.9 for high-quality replicate ChIP samples. For many pairs of replicates, particularly in \emph{D. simulans} and \emph{D. yakuba}, the PCCs were considerably lower and more variable, with some close to 0. Nonetheless, I attempted to call peaks on each matched ChIP-input replicate set using MACS as an exploratory analysis. However, in part due to the different amounts of mapped reads between replicates, some of which were very low, the numbers of peaks called differed widely between replicates (2-605 for \emph{D. melanogaster}, 66-632 for \emph{D. simulans}, 2113-6636 for \emph{D. yakuba} and 105-4458 for \emph{D. pseudoobscura}). Even for the best-matched replicates in terms of numbers of peaks called, which were \emph{D. pseudoobscura} replicates 1 and 3, only 114 peaks overlapped between the two. Visualizing the read densities for these samples confirms the presence of PCR overamplification artefacts and a low degree of reproducibility between biological replicates (Figure 3.8, top three tracks). Interestingly, the input replicates show a greater apparent concordance than the ChIP samples, indicating that the lack of reproducibility in the ChIP samples is not due to the sequencing process alone (Figure 3.8, bottom three tracks). Replicate 3 of the \emph{D. pseudoobscura} ChIP samples also appears to match the input samples better than it matches the other two ChIP replicates, suggesting that the ChIP itself might have failed in this sample.

\begin{figure}
\centering
\includegraphics{fig3-8}
\caption{Dichaete ChIP-seq reads and input reads from three biological replicates in \emph{D. pseudoobscura}. Top three tracks (blue), Dichaete ChIP-seq mapped reads. Bottom three tracks (black), input mapped reads. The same 70-kb region around the gene \emph{slit} is shown as in Figure 3.7. All samples are scaled to 1,000,000 reads for visualization purposes; the y-axis ranges from 0 to 40. Although some similarities are visible between the three ChIP-seq replicates, the input replicates are clearly more reproducible. ChIP-seq replicates 1 and 2 suffer from PCR overamplification in places, as evidenced by sparse coverage and tall, narrow peaks representing highly duplicated reads. ChIP-seq replicate 3 shows more similarity to the input replicates, suggesting that the ChIP reaction might have failed in this replicate.}
\label{Figure 3.8}
\end{figure}

\section{Discussion of results and conclusions}
Confirming previous knowledge about group B Sox proteins in \emph{Drosophila}, my exploration of Dichaete and SoxN in \emph{D. melanogaster, D. simulans, D. yakuba} and \emph{D. pseudoobscura} showed strong evidence for both sequence conservation and functional conservation at the level of spatial and temporal expression patterns. These basic explorations of the orthologous proteins gave me confidence that I could make meaningful comparisons between the binding patterns of each protein in each species without being concerned that their overall functions had diverged too widely. It was also interesting to note that in terms of both sequence and expression pattern, each set of orthologs displayed greater similarity amongst themselves than that displayed by the paralogs Dichaete and SoxN within any one species. In the light of previous data showing partial functional compensation and highly similar binding profiles between Dichaete and SoxN in \emph{D. melanogaster} \citep{ferrero_soxneuro_2014,overton_drosophila_2007}, these data suggest that certain specific, differentiating functions of the two transcription factors have been conserved throughout the evolution of the \emph{obscura} and \emph{melanogaster} groups of drosophilids and are likely ancestral to their divergence \citep{russo_molecular_1995}.\\

One practical purpose of the immunohistochemistry and ChIP-PCR experiments was to determine the suitability of the antibodies raised against the \emph{D. melanogaster} Dichaete and SoxN proteins for performing ChIP-seq against the orthologous proteins in each other species of \emph{Drosophila} studied. In this respect, I needed to determine whether each antibody reacted specifically with each orthologous protein as well as whether it performed well in ChIP reactions with chromatin extracted from each species, as antibodies that work well for immunohistochemistry do not necessarily work well for ChIP. I was able to show that the first question was the case for both Dichaete and SoxN via immunohistochemistry. Embryos from each species stained with each antibody showed highly similar patterns of expression, and background staining was not substantially higher in the non-\emph{melanogaster} species compared to in \emph{D. melanogaster}, indicating that both antibodies react specifically with orthologous proteins from all the species studied.\\

The ChIP-PCR experiments gave more mixed results. In the case of Dichaete, the antibody performed well in \emph{D. melanogaster}, yielding greater enrichment for target sequences in ChIP samples than either mock IP or input samples. Its performance was more variable in other species, although it was unknown whether this was due to the antibody or, as discussed above, the fact that the target sequences were chosen without direct evidence for binding in these species. The SoxN antibody was less successful and did not appear to give significant enrichment of target sequences in any species. Despite the fact that this antibody has been previously used in a ChIP-chip experiment in \emph{D. melanogaster} \citep{ferrero_soxneuro_2014}, I decided not to pursue its use in ChIP-seq in the first instance. Instead, I focused on Dichaete for my initial genome-wide binding experiments. As I was still not completely convinced of the Dichaete antibody's specificity in ChIP experiments, I decided to verify it by first performing a ChIP-chip experiment in \emph{D. melanogaster} and then proceeding to perform ChIP-seq in each species of interest.\\
 
Unfortunately, due to technical problems I was only able to analyze one biological replicate of ChIP-chip data. Although this dataset showed promising enrichment for known Dichaete targets, the results did not have statistical confidence, as I was unable to measure biological or technical variability between samples. My ChIP-seq experiments also suffered from a number of technical problems, most notably contamination by adapter sequences and low library complexity due to PCR overamplification. The results that I was able to generate suggested a higher level of variability between replicate ChIP samples than between replicate input samples. ChIP-chip experiments for Dichaete with the same antibody by a previous lab member were quite noisy; it is my hypothesis that this noise was exacerbated by the greater resolution of ChIP-seq (Aleksic, 2011). Having made these observations, and motivated by decreasing time and budget, I decided that the best course of action was not to pursue further ChIP-seq experiments, but rather to focus entirely on DamID-seq as an alternative method of assaying the genome-wide binding patterns of Dichaete and SoxN.


