\chapter{Chromatin Accessibility During Development in \emph{Drosophila pseudoobscura}}


\section{Experimental Motivation and Design}

Despite having distinct DNA binding domains and preferences for specific sequence motifs, many developmental transcription factors show surprisingly similar genome-wide binding patterns in \emph{D. melanogaster} embryos, differing primarily in quantitative levels of occupancy at a highly-overlapping set of genomic regions \citep{macarthur_developmental_2009}. Both experimental evidence and computational modelling have revealed an important role for chromatin accessibility in determining these overlapping bound regions \citep{kaplan_quantitative_2011,li_role_2011}. Patterns of chromatin accessibility in embryonic nuclei change throughout development as cells take on more committed fates, allowing transcription factors access to different regions of regulatory DNA and ultimately contributing to overall body patterning \citep{thomas_dynamic_2011}. The importance of chromatin accessibility in directing patterns of transcription factor binding has also been observed in \emph{Drosophila} imaginal discs as well as in mammalian cells \citep{john_chromatin_2011,mckay_common_2013,neph_expansive_2012}. Since a major goal of this thesis was to examine differences in transcription factor binding between \emph{Drosophila} species, I was interested in measuring chromatin accessibility during development of non-model drosophilids in order to determine whether observed differences in TF binding could be correlated with differences in accessibility.
   
Two major techniques exist to detect genome-wide patterns of chromatin accessibility in vivo: DNase-seq and FAIRE-seq. DNase-seq relies on the non-specific digestion of chromatin by the enzyme DNaseI. Nuclei are isolated and immediately treated with DNaseI, which cleaves DNA wherever it is accessible. Short DNA fragments resulting from these cleavages are then recovered and sequenced, leading to the identification of DNase-hypersenstive sites (DHS) \citep{thomas_dynamic_2011}. Although this technique has been used extensively, there is some evidence that DHS datasets may suffer from bias due to sequence preferences of DNaseI, which may vary depending on the experimental conditions \citep{koohy_chromatin_2013}. An alternative technique is FAIRE-seq (Formaldehyde-Assisted Identification of Regulatory Elements). In FAIRE-seq, nuclei are isolated and fixed with formaldehyde. The chromatin is then sonicated, breaking the more accessible regions into small fragments, and purified using phenol-chloroform extractions. This results in only DNA from accessible regions being recovered, as inaccessible, compacted chromatin is left in the organic phase during the extractions \citep{giresi_isolation_2009,simon_using_2012}. Although DNase-seq and FAIRE-seq do not perfectly recapitulate each other, as DNAse-seq tends to detect a higher signal at promoter regions while FAIRE-seq tends to detect a higher signal at distal regulatory regions, overall the two techniques show quite good correspondence \citep{koohy_chromatin_2013,mckay_common_2013}.
   
I decided to use FAIRE-seq to study chromatin accessibility and to focus on one species, \emph{D. pseudoobscura}, which is the most distant species to D. melanogaster of those that I studied and which shows the greatest difference in chromosomal structure and arrangement. %explain D. pseudo karyotype 
I performed FAIRE-seq on \emph{D. pseudoobscura} embryonic chromatin from five developmental stages, stage 5, stage 9, stage 10, stage 11 and stage 14, chosen to provide a comparison with \emph{D. melanogaster} DNase-seq data from Thomas et al. (2011) \nocite{thomas_dynamic_2011}. I sequenced three biological replicates from each stage. A detailed description of the methods used in the FAIRE protocol and for processing the sequencing data can be found in Chapter 2. Although input chromatin can be used as a control for FAIRE-seq, as with ChIP-seq, it is not strictly necessary \citep{simon_using_2012}. Indeed, as one of the sources of the non-random patterns of reads observed in input controls is chromatin accessibility, it is possible that using such a control with FAIRE-seq would reduce the detection of true FAIRE signal. For my FAIRE-seq experiments, I did not sequence matched input controls for each developmental stage, but rather used GC-content and mappability data calculated from the \emph{D. pseudoobscura} genome to correct for potential biases in the data during analysis.

\section{FAIRE-seq results}

\section{Comparison with chromatin accessiblity data in \emph{D. melanogaster}}

\hrulefill


