\chapter{Discussion and Future Directions}

\hrulefill

\section{Regulatory function and evolution}
In their rebuttal to the conclusions of the ENCODE consortium, Graur and colleagues write that "[f]rom an evolutionary viewpoint, a function can be assigned to a DNA sequence if and
only if it is possible to destroy it ... Unless a genomic functionality is actively protected by selection, it will accumulate deleterious mutations and will cease to be functional \citep{graur_immortality_2013}." According to this definition of function, a transcription factor binding site and, by extension, a TF binding event, is functional not simply because it occurs but if it has a result that can be altered or broken by its loss. The classical, and most stringent, way to detect such functional binding events is to combine genome-wide studies of \emph{in vivo} binding patterns with gene expression data in a mutant background to detect genes that are both bound by a TF and change expression levels upon its loss or overexpression. This approach has yielded fruitful results in the past with both \emph{Dichaete} and \emph{SoxN} in \emph{D. melanogaster} \citep{aleksic_role_2013,ferrero_soxneuro_2014,shen_identifying_2013}. However, it can also be an overly conservative strategy, since not all TF functions result in a direct change in expression of the nearest gene, particularly for TFs like Sox proteins that can bend DNA and potentially alter the local chromatin environment \citep{bowles_phylogeny_2000,ferrari_sry_1992,giese_hmg_1992,russell_dichaete_1996}. Additionally, the effects of the loss of one particular binding site can be masked by robustness from secondary shadow enhancers or other members of the regulatory network, particularly in the relatively stress-free lab environment \citep{aldana_robustness_2007,ciliberti_innovation_2007,ludwig_consequences_2011,perry_shadow_2010}. In this thesis, I have attempted to focus on the second part of Graur \emph{et al}.'s definition, using the conservation of TF binding during evolution as a filter through which to refine our understanding of group B Sox function in \emph{Drosophila}. In this final chapter, I will review the major findings from my analysis, present a model for SoxN and Dichaete binding that arises from the evolutionary patterns I have observed, and speculate on the origin of both redundancy and neofunctionalization between Sox genes in the vertebrate and invertebrate phylogenies.

\section{Major conclusions of experimental results} 
As outlined in the introduction, I set out to study the conservation of group B Sox function on several functional levels, ranging from the DNA sequence of target regions to expression patterns and overall phenotypic effects. Starting from the highest level, I found that the roles of Dichaete and SoxN within the fly developmental regulatory network do not appear to have diverged significantly during the evolution of the \emph{Drosophila} species examined. The gene targets and genomic annotations associated with Dichaete and SoxN function are largely conserved between \emph{D. melanogaster, D. simulans, D. yakuba} and \emph{D. pseudoobscura}, which is not surprising given the high degree of sequence similarity between the orthologous proteins in each species and their equivalent expression patterns during embryonic development. However, for both transcription factors, a comparison of \emph{in vivo} binding patterns revealed turnover of binding sites at gene loci as well as quantitative divergence in binding affinity between species. In the case of Dichaete, for which binding was compared between four species, the proportion of \emph{D. melanogaster} binding intervals that are not conserved in each other species increases with phylogenetic distance. This observation is in line with previous comparative studies of other transcription factors in \emph{Drosophila} and suggests that, as with other DNA binding proteins, the evolution of group B Sox binding may follow a molecular clock mechanism \citep{bradley_binding_2010,he_high_2011,paris_extensive_2013}. The range of binding divergence at the evolutionary scale studied, which is less than that between vertebrate species compared in similar studies with other TFs \citep{odom_tissue-specific_2007,schmidt_five-vertebrate_2010,stefflova_cooperativity_2013,villar_evolution_2014}, enabled me to identify patterns of increased conservation compared to the background rate at certain functional categories of binding interval.\\

As expected, group B Sox binding is highly conserved at sites that are most likely to be involved in functional gene regulation, including known enhancers from the REDFly and FlyLight databases \citep{gallo_redfly_2010,manning_resource_2012}, Dichaete and SoxN direct target genes, and the Dichaete and SoxN core binding intervals believed to represent very high confidence \emph{in vivo} binding locations \citep{aleksic_role_2013,ferrero_soxneuro_2014}. These findings validate the hypothesis that a signature of selective constraint in the form of increased conservation can be found at functional sites. They also confirm that the Dichaete and SoxN binding events identified in multiple \emph{in vivo} genome-wide studies are functionally important, whether through direct transcriptional regulation or an indirect architectural role \citep{russell_dichaete_1996}. Interestingly, binding at core intervals was shown to be more highly conserved than binding at direct target genes, suggesting that binding site turnover can occur even at direct targets. Integrating the FAIRE-seq chromatin accessibility data with the DamID-seq data reveals that not only is group B Sox binding associated with open chromatin in multiple species of \emph{Drosophila}, binding in accessible chromatin is more likely to be conserved between species. This relationship likely reflects a feedback loop whereby chromatin accessibility patterns direct transcription factor binding and selection on functionally bound enhancer elements works to maintain open chromatin.\\

One of the primary questions of my work was whether common binding by Dichaete and SoxN is conserved to the same extent as specific binding by each protein at unique targets. A comparative analysis of DamID for both TFs in \emph{D. melanogaster} and \emph{D. simulans} revealed that, in fact, common binding is much more likely to be conserved than unique binding by either Sox protein. This is true from the perspective of \emph{D. melanogaster} binding intervals that are conserved in \emph{D. simulans} as well as vice versa. Such a high rate of conservation of common binding strongly suggests that the ability of Dichaete and SoxN to bind to and regulate a set of common targets and to compensate for each other at those targets is an important aspect of their biological function. The targets of commonly bound, conserved binding intervals reflect the known functions of group B Sox proteins in the developing central nervous system. Target genes are primarily upregulated in the CNS, and they are enriched for Gene Ontology terms related to biological regulation, morphogenesis, and the specification and differentiation of neurons. They also include targets where Dichaete and SoxN have previously been shown to demonstrate compensation, such as the homeodomain DV-patterning genes \emph{ind} and \emph{vnd}, as well as targets where Dichaete and SoxN appear to have opposite regulatory effects, such as \emph{ac} and \emph{l'sc}, both proneural genes, or \emph{pros}, a TF involved in neuroblast differentiation \citep{aleksic_role_2013,ferrero_soxneuro_2014,overton_evidence_2002}.\\

\emph{D. melanogaster} and \emph{D. simulans} also share smaller numbers of conserved binding intervals that are uniquely bound by either Dichaete or SoxN. The primary difference in the target genes annotated to these intervals is in their expression profiles. Uniquely bound Dichaete targets show expression in a broader range of tissues, including the brain and hindgut, where Dichaete is known to play a role \citep{sanchez-soriano_regulatory_2000}, while uniquely bound SoxN targets show strong upregulation only in the developing CNS. Conserved binding regions unique to Dichaete also contain a highly enriched motif for Byn, a transcription factor that is necessary for hindgut development, which may represent a new physical or genetic interaction specific to Dichaete in the hindgut \citep{kispert_homologs_1994,murakami_developmental_1999}. Although unique SoxN targets have a similar expression profile as common targets, other features of these targets, including their enrichment in the Robo-Slit signalling pathway and the presence of an enriched Usp motif in binding intervals, indicate that they may play important and unique roles in axon guidance. This confirms the functional importance of a number of previously discovered SoxN targets involved in later stages of neuronal differentiation as well as the large overlap observed between SoxN targets and fly orthologues of targets of mouse Sox11, a group C Sox protein primarily expressed in differentiated neurons \citep{bergsland_sequentially_2011,ferrero_soxneuro_2014}. These features of unique Dichaete and SoxN binding are more clearly apparent when analyzing data from two species, rather than from \emph{D. melanogaster} alone. In the initial comparison of Dichaete and SoxN binding in \emph{D. melanogaster} and \emph{D. simulans}, it appeared that the two TFs had more differentiated binding patterns in \emph{D. simulans}. Using evolutionary conservation as a filter may have reduced the effect of noise in these datasets, allowing me to home in on the truly unique functions of each protein.\\

One interesting effect of the use of a quantitative analysis of binding differences between TFs is that it allowed me to identify both a subset of genes that are uniquely bound by Dichaete or SoxN in multiple species and a subset of genes that are preferentially bound by each TF. These preferential targets show binding at the same regions of regulatory DNA by both Dichaete and SoxN across species, but they are consistently bound by one protein at a higher affinity than by the other. Many of these preferentially bound targets are identified as common targets of Dichaete and SoxN in a qualitative analysis of binding. Although for both Dichaete and SoxN, the preferential targets have similar expression profiles as the unique targets, the lists of preferentially and uniquely bound genes show relatively low overlap (76 genes for SoxN and 169 for Dichaete). The preferentially bound genes in each case may highlight binding sites where the regulatory function of Dichaete and SoxN has diverged, but their HMG domains remain similar enough that they can both recognize and bind to the same DNA sequences, particularly under the conditions of DamID, when both proteins are expressed uniformly at comparable levels. Preferential targets include genes whose regulation has been shown to be important for Dichaete and SoxN function, including \emph{pros} in the case of Dichaete and \emph{ase, ind} and \emph{vnd} in the case of SoxN \citep{aleksic_role_2013,ferrero_soxneuro_2014,overton_evidence_2002}. These may also represent cases where Dichaete and SoxN can compensate for one another to increase the robustness of key regulatory networks.\\

A sequence-based analysis of the Sox motifs found in each set of DamID binding intervals revealed some subtle differences in the binding motifs preferred by each TF, primarily at position six of the consensus A/TA/TCAAAG motif. Previous studies have indicated that this nucleotide is more likely to be a thymine residue in Dichaete core intervals and an adenine residue in SoxN core intervals; however, it was not known whether this difference reflected any underlying differences in the structures of the two proteins \citep{aleksic_role_2013,ferrero_soxneuro_2014}. The DamID approach employed here used the same fusion proteins, derived from the Dichaete and SoxN sequences in \emph{D. melanogaster}, to assess binding in all species, meaning that binding differences due to evolutionary changes in orthologous proteins could not be detected. However, multiple alignments of the amino acid sequences show that there is a higher degree of sequence conservation, including a perfectly conserved HMG box domain, between each set of orthologous proteins than between Dichaete and SoxN in any one species. Consequently, it should be easier to detect potential differences in motif preference between paralogues than between orthologues. The fact that very similar differences in the Sox motifs for Dichaete and SoxN were found independently in each genome studied indicates that these sequence preferences are likely to be real and may reflect differences in the preferred binding modes of each protein.\\

Considering all variants of the Sox motif detected, intervals which show conserved binding in all four species contain more motifs on average than those that are only bound in one species; these motifs are also more highly conserved at both the nucleotide level and at the level of positional organization within regulatory regions. It should be noted that I did not perform any classical tests for selection on either the binding interval or Sox motif sequences. This is partly because, although methods such as the McDonald-Kreitman test have been adapted for use with non-coding DNA, it is difficult to establish an appropriate neutral reference against which to test for selection in putative functional sites \citep{anisimova_methods_2012}. Testing for selection in entire enhancers is difficult because, while high-confidence TF binding sites may be identified, it is often unknown whether the majority of the sequence is functional or not. Detecting selection at specific motifs or binding sites is more feasible, and alternative methods have been proposed to do so \citep{moses_statistical_2009}; however, such tests still rely on the presence of substitutions and polymorphism, which were not found in many of the Sox motifs that I uncovered. Although I was unable to detect an effect of motif quality on quantitative binding affinity, the finding that Sox motifs in intervals that show binding conservation also show increased rates of conservation provides a link between group B Sox function and DNA sequence evolution, as well as a mechanism through which natural selection can act to maintain functional binding.  

\section{Toward a selection-based model of group B Sox binding}
One of the primary findings of this work is the high rate of evolutionary conservation of Dichaete and SoxN binding at sites where both proteins can bind compared to sites where only one protein is bound in vivo. The implication of this is that such common binding is an important feature of group B Sox function. Many of the potential target genes annotated to these intervals are known Sox targets in the developing CNS; one hypothesis as to why these binding sites might be preferentially conserved is to confer robustness on cell fate decisions from the specification of the neuroectoderm through to neuroblast differentiation, axonogenesis and gliogenesis \citep{ferrero_soxneuro_2014,wagner_distributed_2005,wagner_gene_2008}. This is supported by the partial functional redundancy between \emph{Dichaete} and \emph{SoxN} seen on a phenotypic level in single mutant embryos, as well as at certain loci where one protein can substitute for the binding of another in its absence \citep{ferrero_soxneuro_2014,overton_evidence_2002}. However, increased conservation is also seen at binding sites where Dichaete and SoxN have opposite regulatory functions, several of which are also critical for determining neuroblast fate \citep{aleksic_role_2013,ferrero_soxneuro_2014,overton_evidence_2002}. Why would natural selection preferentially maintain binding of factors with antagonistic functions at the same sites?\\

It is possible that in some situations, Dichaete and SoxN might directly compete with one another for binding. Although this has not been demonstrated, it is a feasible mechanism for establishing a balance between the up- or down-regulation of genes promoting neuroblast differentiation, for example, with the ultimate outcome dependent on the relative concentration of each TF within a cell. All of the group B Sox proteins have very similar HMG domains and can recognize similar consensus DNA sequences \citep{mckimmie_conserved_2005}, despite the discovery in this study of some possible differences in motif preference between Dichaete and SoxN. Another view for the explanation of common binding is simply that it is easier for natural selection to maintain Sox motifs in enhancers that can be bound by both TFs than to maintain a suite of slightly different motifs for each. Analogous to a gene duplication event, one might expect that a newly originated TF binding site would often experience low selective pressure and quickly accumulate mutations, resulting in the maintenance of a minimal complement of sites. Such a mechanism could be self-reinforcing, as sites that are functionally bound by multiple TFs would experience a higher dose of selective constraint since mutations that disrupted binding would perturb the regulatory networks associated with both TFs. This could also explain the fact that group B Sox proteins have primarily diversified in regions other than their DNA-binding domains during evolution; strong selection on common binding sites would encourage the acquisition of new functions through interactions with specific binding partners or changes in the ability to modify the local chromatin environment.\\

Given the observed selective constraint on commonly bound sites, what is the explanation for the presence of highly conserved sites that are uniquely bound by either SoxN or Dichaete? The fact that target genes at these sites have different spatial expression profiles suggests a model whereby the different expression patterns of Dichaete and SoxN themselves, along with extrinsic factors in the nuclear environment, may shape the unique functions of these two TFs. Although \emph{Dichaete} and \emph{SoxN} expression patterns overlap to a great extent in the CNS, they are not identical; \emph{Dichaete} is expressed uniquely in the midline, brain and hindgut, for example, while \emph{SoxN} is expressed uniquely in the lateral column of delaminating neuroblasts and shows specific patterns of expression in the epidermis at later stages of development \citep{cremazy_<_2000,overton_drosophila_2007,sanchez-soriano_regulatory_2000,soriano_drosophila_1998}. The chromatin landscape has been shown to differ between different tissues in the \emph{Drosophila} embryo as well as over the course of development, both in terms of general accessibility and specific activating or repressing histone marks \citep{bonn_tissue-specific_2012,mckay_common_2013}. It is therefore likely that certain enhancers are only available to be bound in the tissues where Dichaete and SoxN are expressed uniquely, preventing common binding from ever being observed. A comparative analysis of chromatin accessibility and histone marks between the hindgut and the CNS would be a fascinating way to test this hypothesis with regard to unique and common binding by Dichaete in these tissues.\\

Another possible factor that could explain the unique, conserved binding patterns of Dichaete and SoxN in different tissues is the tissue-specific presence of certain cofactors. Sox proteins often bind to DNA as heterodimers with other TFs \citep{ambrosetti_synergistic_1997,archer_interaction_2011,bery_characterization_2013,bonneaud_sncf_2003}; Dichaete has previously been demonstrated to bind together with Vvl in the midline \citep{ma_functional_2000,soriano_drosophila_1998}. As discussed in the introduction, although paralogous Hox proteins generally show greater specificity in their gene targets than Sox, it has been suggested that much of this specificity may arise from interactions with binding partners \citep{chan_switching_1997,mann_chapter_2009,slattery_genome-wide_2011}. The sequences of \emph{Drosophila} group B Sox genes have diverged much more outside of their HMG domains than within them, although sections of the C-terminal regions of both Dichaete and SoxN still show good levels of conservation between fly species, suggesting that a major driver of their evolutionary diversification may have been the acquisition of new cofactors \citep{mckimmie_conserved_2005}. Although changes in target specificity due to binding with cofactors has not been demonstrated for Sox proteins in \emph{Drosophila}, this is a feasible mechanism behind the specific binding of Dichaete and SoxN in different embryonic tissues. The identification of enriched motifs in Dichaete- and SoxN-specific binding intervals that are not present in common binding intervals, corresponding to TFs such as Byn in the case of Dichaete and Usp in the case of SoxN, may represent tissue-specific cofactors of these proteins, although physical interactions remain to be demonstrated.\\

The patterns of conservation of binding by Dichaete and SoxN in \emph{Drosophila} suggest a model whereby, despite slight differences observed in the consensus motifs bound by each protein, both group B Sox proteins can and do bind a majority of their sites in common in those tissues where they are both expressed. These common binding sites are preferentially maintained during evolution in comparison to sites that are uniquely bound, whether to increase the robustness of the regulatory networks shared by Dichaete and SoxN or due to the effect of selection favoring the re-use of binding sites in a dense, compact genome. At the same time, unique binding by Dichaete and SoxN is conserved at specific target genes that have largely different expression profiles, possibly reflecting the effect of tissue-specific chromatin landscapes or cofactor availability. These observations are supported by the fact that the majority of the sequence differences between Dichaete and SoxN can be found outside of their HMG domains, in protein domains that may be involved in interactions with other TFs as well as in their own regulatory regions, which determine both the overlapping and unique expression patterns of each TF.

\section{Implications for the evolution of Sox function and redundancy}
\emph{Sox} genes encode an ancient family of transcription factors that, despite numerous gene duplication events, continue to show functional redundancy between members of the same subgroups throughout their phylogeny \citep{bhattaram_organogenesis_2010,ferri_sox2_2004,guth_having_2008,matsui_redundant_2006,nishiguchi_sox1_1998,okuda_b1_2010,overton_evidence_2002,rizzoti_sox3_2004,uchikawa_b1_2011,uwanogho_embryonic_1995,wegner_stem_2005,wood_comparative_1999}. While common binding patterns between two TFs do not necessarily imply redundancy or compensation, they are, if not required for it, then likely to facilitate it. Indeed, SoxN and Dichaete have a complex relationship in the fly embryo that includes functional compensation as well as interdependence and binding at some loci with opposite regulatory effects \citep{ferrero_soxneuro_2014,overton_evidence_2002}. Previous \emph{in vivo} binding studies as well as studies of gene expression changes in mutants have identified large numbers of genes bound in common by Dichaete and SoxN \citep{aleksic_role_2013,ferrero_soxneuro_2014}. This study shows that such common binding has been conserved during the evolution of the drosophilids at a rate higher than that of unique binding by either protein, suggesting that it is a key feature of group B Sox function. It has been speculated that robustness may arise as a general property of complex gene regulatory networks, without any direct selective pressure \citep{aldana_robustness_2007}. My results show that, in the case of \emph{Drosophila} group B Sox genes, partially redundant binding patterns that can lead to increased robustness may also be specifically maintained by natural selection.\\

Given the apparently independent evolutionary trajectories of group B \emph{Sox} genes in insects and mammals and the fact that it is difficult to assign direct orthology between individual members of each class, an obvious question is whether the partial redundancy seen among group B \emph{Sox} genes in mammals is a shared ancestral feature or the result of convergent evolution \citep{bergsland_sequentially_2011,ferri_sox2_2004,nishiguchi_sox1_1998,okuda_b1_2010,rizzoti_sox3_2004}. Although the evolutionary models proposed by McKimmie \emph{et al}. and Zhong \emph{et al}. differ, they both suggest that at least one tandem duplication event occurred at the ancestral group B \emph{Sox} locus before the protostome/deuterostome split (Figure 7.1) \citep{mckimmie_conserved_2005, zhong_parallel_2011}. According to Zhong and colleagues, this duplication gave rise to the proto-B1 and -B2 genes, which then expanded in the vertebrates through whole-genome duplications and in the arthropods through further tandem duplications \citep{zhong_parallel_2011}. This model can account for the divergent functions seen in vertebrate group B1 and B2 genes \citep{uchikawa_two_1999}; however, it does not explain the ability of both \emph{Dichaete} and \emph{SoxN} to play B1-like and B2-like roles. If the pattern seen in the vertebrate \emph{Hox} gene expansions, in which \emph{trans}-paralogues arising from genome duplications show greater functional similarity than co-linear \emph{cis}-paralogues, holds any general applicability, then the McKimmie \emph{et al}. model may be more consistent with functional data. In this model, \emph{Dichaete} and \emph{SoxN} ancestors arose via whole-genome duplication followed by a tandem duplication to create the \emph{Sox21a}/B2-like ancestor, both prior to the protostome/deuterostome split. In the arthropod lineage, a further tandem duplication led to the origin of \emph{Sox21b}, while in the vertebrate lineage, another genome duplication event filled out the complement of group B1 and group B2 genes \citep{mckimmie_conserved_2005}. If this is the case, then redundancy between the first group B paralogues resulting from genome duplication may have been partially retained throughout evolution, while later paralogues split into group B1 and B2 functions in vertebrates or acquired partial neofunctionalizations in insects.\\

\begin{figure}
\centering
\includegraphics{fig7-1}
\caption{Two models of the evolution of group B \emph{Sox} genes in vertebrates and insects. A.) The model proposed by McKimmie \emph{et al.} In this model, an ancestral group B \emph{Sox} gene gave rise to the \emph{Dichaete} and \emph{SoxN} ancestors via a whole-genome duplication. The \emph{Sox21a} ancestor then arose through a tandem duplication before the protostome/deuterostome split. Finally, a further tandem duplication generated \emph{Sox21b} in the insect lineage, while another whole genome duplication led to the origin of the remaining group B \emph{Sox} genes in vertebrates. B.) The model proposed by Zhong \emph{et al}. In this model, a single tandem duplication before the protostome/deuterstome split gave rise to the ancestral \emph{SoxB1} and \emph{SoxB2} genes. These then underwent two rounds of whole genome duplications in vertebrates to generate the full complement of group B \emph{Sox} genes, while in insects two further tandem duplications led to the origin of \emph{Sox21a} and \emph{Sox21b}. Figure reproduced from \citet{zhong_parallel_2011}.}
\label{Figure 7.1}
\end{figure}

Such a model suggests that functional redundancy between group B \emph{Sox} genes, particularly in the developing CNS, is a truly ancestral feature that has been refined and elaborated upon separately in different lineages. In vertebrates, multiple genome duplicates have given rise to a larger complement of \emph{Sox} genes, which have apparently undergone a greater degree of subfunctionalization and neofunctionalization, while still retaining overlapping expression patterns and some degree of functional compensation. This particularly appears to be true of group B \emph{Sox} genes, whose function in the CNS can be split both by temporal succession \citep{bergsland_sequentially_2011} and by activator/repressor roles \citep{uchikawa_two_1999}. While it is somewhat surprising that substantial functions in the CNS have not been discovered for \emph{Sox21a} and \emph{Sox21b} in \emph{Drosophila}, this underscores the observation that, in insects, \emph{Dichaete} and \emph{SoxN} appear to direct virtually all aspects of neurogenesis and are the only \emph{Sox} genes to do so \citep{ferrero_soxneuro_2014}. Although \emph{Dichaete} and \emph{SoxN} have undoubtedly undergone partial neofunctionalization, which is reflected both in their expression patterns and in their unique binding targets, they have maintained a close and complex relationship comprising aspects of interdependence, antagonistic regulatory effects and compensation. If the integrated action of multiple Sox proteins, whether as opposing factors or to provide additional robustness, is a feature of CNS development that has been consistently selected for, it is possible that the presence of additional group B Sox proteins in vertebrates has led to a relaxation of this selective pressure and allowed them to specialize to a greater degree. Although unique binding by Dichaete and SoxN in \emph{Drosophila} is less conserved than common binding, it still occurs at numerous loci throughout the genome. These binding events, while not necessarily functional, may represent opportunities for further neofunctionalization through an unconstrained exploration of the regulatory landscape.

\section{Future work}
Although this thesis has shed some light on the conserved functional relationship between \emph{Dichaete} and \emph{SoxN} in several species of \emph{Drosophila}, the complex functions of group B \emph{Sox} genes in invertebrates are far from completely understood. A number of experimental approaches could help to validate the model proposed in this thesis. The major drawbacks of DamID include its lack of tissue specificity and the fact that it does not measure TF binding in its native context, but rather by using a transgenic fusion protein expressed in addition to the endogenous protein. Using ChIP as a complementary technique can help reduce these problems; since each technique is subject to different sources of bias, a binding dataset derived from intersecting the two will be much more stringent than using either technique alone \citep{aleksic_role_2013}. However, given the lack of success in performing ChIP using current antibodies for Dichaete and SoxN, this does not appear to be the most promising avenue for further research, unless new and more reliable antibodies for group B Sox proteins in insects can be derived. Fortunately, a targeted DamID technique (TaDa) has recently become available, which enables the measurement of binding in specific tissue or cell types \citep{southall_cell-type-specific_2013}. Using TaDa to dissect Dichaete and SoxN binding patterns in tissues where they are commonly expressed, such as the medial and intermediate columns of neuroblasts, versus tissues where only one is present, such as the hindgut or midline, would be a useful follow-up both to identify tissue-specific enhancers and target genes and to test whether unique binding is indeed primarily driven by tissue-specific factors. This could also provide \emph{in vivo} binding data with a greater temporal resolution, as the expression patterns of Dichaete and SoxN change throughout developmental time, which could then be correlated with the detailed time course of gene expression changes in \emph{Dichaete} and \emph{SoxN} mutant backgrounds that is already available \citep{ferrero_soxneuro_2014}.\\

The proposed sources of tissue-specific binding, namely chromatin accessibility and the presence of specific cofactors, could also be tested. The two primary techniques for assessing chromatin accessibility, DNase-seq and FAIRE-seq, can both feasibly be applied in dissected tissues, although FAIRE-seq has been more effective in this regard in \emph{Drosophila} because embryos or larvae can be fixed before dissection, greatly facilitating the process of collecting material \citep{mckay_common_2013}. For tissues that cannot easily be dissected, BiTS-ChIP, a technique involving fluorescently sorting fixed nuclei that are tagged with a cell-type specific marker \citep{bonn_tissue-specific_2012}, could be used to study chromatin accessibility either in combination with FAIRE or by performing ChIP for a general marker of transcriptional activity such as RNA polymerase II (Pol II). Similarly, performing TaDa with a Pol II-Dam fusion protein can also yield data on cell-type specific chromatin landscapes \citep{southall_cell-type-specific_2013}. The use of these techniques in tissues where Dichaete and SoxN are commonly or uniquely expressed would enable the discovery of enhancers or regulatory regions that are only accessible in certain tissues, which could then be correlated with binding patterns. In order to determine whether tissue-specific cofactors can direct Dichaete and SoxN binding, \emph{in vitro} methods such as co-immunoprecipitation could be used to test for physical interactions between group B Sox proteins and potential cofactors. \emph{In vivo}, the dependence of group B Sox binding on candidate cofactors could be tested by measuring Dichaete or SoxN binding, either through ChIP or DamID, in a mutant background for the cofactor of interest. If performed in a tissue-specific manner, this experiment could yield convincing data either in support of or against a model whereby Dichaete and SoxN have acquired unique binding sites through interactions with other TFs that are only present in a subset of their spatial expression domains.\\

On the level of individual targets, a large number of instances of putative binding site turnover events have been identified for Dichaete and SoxN, where non-orthlogous regulatory regions for the same gene are bound in vivo in different species of \emph{Drosophila}. It is hypothesized that such turnover is subject to constraint such that, in the absence of a gain of function driven by positive selection, the overall level of gene expression should be buffered \citep{he_does_2011,spivakov_analysis_2012}. This hypothesis could be tested by performing reporter assays in transgenic lines of \emph{D. melanogaster} carrying putative enhancer sequences from other species \citep{hare_sepsid_2008}. Although a staining-based assay would not provide a quantitative measure of gene expression, it would allow for the detection of any differences in spatial expression patterns driven by species-specific enhancer elements.\\

In order to further refine our understanding of group B \emph{Sox} function and evolution, it would be useful to expand the work done here into species more distant from \emph{D. melanogaster}. Such a project is currently underway in the red flour beetle, \emph{Tribolium castaneum}. It is not currently known whether the fifth group B \emph{Sox} gene present in \emph{Tribolium}, \emph{SoxB3}, is functional or represents a pseudogene; its expression pattern has not yet been determined. If it is functional and expressed in the developing CNS, it could add yet another layer of complexity and potential compensation to the functional roles of group B \emph{Sox} genes. However, sequence analysis suggests that, if \emph{Tribolium SoxB3} is functional, it may have diverged sufficiently from its paralogues to have acquired a new, independent function. If so, then it would represent an exceptional case of neofunctionalization in the insect \emph{Sox} clade. Although it becomes progressively more difficult to align genomes as the phylogenetic distance between two species being compared increases, complicating the assignment of orthology to putative enhancer regions and binding events, such a comparison has the potential to reveal stronger selective effects and more deeply conserved features of TF binding. The use of a more distant \emph{Drosophila} species or another non-\emph{Drosophila} dipteran whose genome is available, such as the scuttle fly \emph{Megaselia}, would also be very useful in this regard, as it would enable a comparison of binding patterns in the context of greater sequence divergence but highly conserved mechanisms of embryonic patterning and development \citep{hare_sepsid_2008}.\\

Other experiments that would help to progress this work include a more detailed dissection of Dichaete and SoxN binding sites as well as an exploration of other factors that interact with Dichaete and SoxN in the \emph{Drosophila} transcriptional regulatory network. Both DamID and conventional ChIP-seq have sufficient resolution to identify binding events on the scale of a few hundred base pairs, but as described in this thesis, those binding intervals often contain multiple matches to a TFâ€™s consensus binding site. Particularly for DamID, it is difficult to identify the actual DNA sequence to which the TF is bound \emph{in vivo}. Even with ChIP-seq data, it can be difficult to distinguish between a single bound motif and multiple, closely-spaced bound motifs. ChIP-exo, in which ChIP DNA is treated with an exonuclease to digest away non-bound nucleotides before sequencing, is a technique that can help overcome these limitations and identify bound sites with very high resolution \citep{bardet_identification_2013,rhee_comprehensive_2011}. Although it is also antibody-dependent, it would be very interesting to perform ChIP-exo for Dichaete and SoxN in multiple species of \emph{Drosophila}, as it would enable a much more detailed comparison of the binding sites preferred by each protein and the evolutionary forces to which they are subject.\\

In addition to identifying new, unique cofactors for Dichaete and SoxN, it would also be informative to study factors with which they have already been suggested to interact in an evolutionary context. A comparative analysis of Dichaete binding intervals and binding profiles of 33 other TFs in \emph{D. melanogaster} identified seven TFs whose profiles significantly overlapped with that of Dichaete. Four of these, Senseless (Sens), Prospero (Pros), Hunchback (Hb), and Kruppel (Kr), are known to be involved in CNS development \citep{aleksic_role_2013}. Hb and Kr are the first two transcription factors expressed in a temporal series in embryonic neuroblasts as they differentiate into ganglion mother cells (GMCs), during the time that both Dichaete and SoxN are expressed in the developing neuroectoderm \citep{buescher_formation_2002,maurange_brainy_2005,overton_evidence_2002}. Since \emph{hb} and \emph{Kr} were found to be targets of both Dichaete and SoxN in this and previous studies \citep{aleksic_role_2013,ferrero_soxneuro_2014}, they may form part of a feed-forward loop with the group B \emph{Sox} genes in the genetic regulatory network specifying neuroblast fate. Assessing the conservation of overlaps between Dichaete, SoxN, Hb and Kr binding patterns in multiple species of \emph{Drosophila} could help clarify the targets that are commonly regulated by these factors.\\

Finally, in order to paint a complete picture of the evolution of insect \emph{Sox} genes and their roles in development, it will be necessary to address the functions of the remaining two group B \emph{Sox} genes, \emph{Sox21a} and \emph{Sox21b}. While \emph{Sox21a} is expressed in the midline as well as the anlage of the foregut and hindgut, \emph{Sox21b} is excluded from the CNS but is expressed in the ventral epidermis and the hindgut, where it overlaps with \emph{Dichaete} expression \citep{cremazy_genome-wide_2001,mckimmie_conserved_2005,phochanukul_no_2010}. Surprisingly, deletions of either of these genes individually or both together produce no observable phenotype in \emph{D. melanogaster}. However, they show conservation at both a sequence level and in terms of genomic location across the insects, suggesting that they do provide some functionality \citep{mckimmie_conserved_2005}. Perhaps their role is largely limited to increasing the robustness of the \emph{SoxN}- and \emph{Dichaete}-driven regulatory networks in specific cell types, although it would be surprising if they had no independent functions and yet remained conserved. Additionally, the only other \emph{Sox} gene known to be expressed in the developing CNS in \emph{Drosophila} is \emph{Sox102F}, the only insect group D \emph{Sox} gene \citep{cremazy_genome-wide_2001,phochanukul_no_2010}. Although RNAi-mediated knockdown of \emph{Sox102F} results in severe CNS disruptions, its gene targets are not known \citep{phochanukul_no_2010}. Genome-wide \emph{in vivo} binding studies of these three Sox proteins would help to fill in the gaps in our current knowledge of insect \emph{Sox} biology and possibly provide new data on functional compensation by Sox proteins in the developing CNS.

\section{Conclusions}
Genetic redundancy is a curious phenomenon because it appears to violate the rule that a biological function exists if and only if it can be broken \citep{graur_immortality_2013}. Indeed, redundancy among \emph{Sox} genes was first described by researchers who were no doubt frustrated by observing that single mutants generated in model animals appeared phenotypically normal. Nonetheless, it has been observed throughout the \emph{Sox} family tree, in multiple subgroups and species \citep{bhattaram_organogenesis_2010,ferri_sox2_2004,matsui_redundant_2006,nishiguchi_sox1_1998,okuda_b1_2010,rizzoti_sox3_2004}. The hypothesis that redundancy can confer robustness on a genetic regulatory network suggests that redundancy itself may be a function that can be broken \citep{nowak_evolution_1997,tautz_redundancies_1992,wagner_distributed_2005,wagner_gene_2008}. While it does appear that compensation between Dichaete and SoxN may lend robustness to the developing CNS, their relationship is clearly much more complex than one of simple redundancy \citep{ferrero_soxneuro_2014}. Evolutionary comparisons reveal that this relationship is conserved across species and that the functions of Dichaete and SoxN binding are intimately tied to one another. I hope that through this work I have demonstrated the value of studying transcription factor binding patterns through the lens of natural selection while refining the current model of the common and unique functions of group B \emph{Sox} genes in \emph{Drosophila} development.\\

